initial-data-load/data/Sample/MapLayers.yml

```diff
@@ -14,13 +14,6 @@
   legend:
     - type: "colormap"
       color: "blue"
-  metadata:
```

Comment:
Hi, these metadata are moved to GeoData.yml because they are about geojson not attributes. Now no metadata are left in MapLayers.yml. Should we show on frontend geoMetadata from GeoData.yml  instead of them? Or it doesn't matter for now? Or I can put:
    description: "Sample data created just for this purpose. Do not represent the reality in any time."
    updateFrequency: "never"
    unit: "n/a"
    dataRetrievalDescription: "Data was randomly created."
    
also into MapLayers.yml  metadata to have example there.

Reply:
@bariela can you maybe help here? To me some of the metadata are connected with geodata, and some might be connected with map layer

Reply:
@bariela Hi, can you please check this so we can merge it? Thanks a lot

Reply:
Hi,
each data/metadata should be only in one place and before returning to frontend, they should all be merged into one object. And yes, only some metadata of are linked with geoData. Did this answer the question?

Reply:
@MartinJurcoGlina @bariela Hi, I refactored this one after consultation with Daniela, now its returning geoMetadata in Maplayers metadata. It is working but I can't see, when I click on info icon the metadata. Sorry I don't have time to fix this, but it should be easy. I will leave it to you. Also function failed in pipeline but I don't know why. I think Daniela had some problem with them too.